scarlet/blend.py

```diff
@@ -262,8 +262,9 @@ def _loss_func(self, *parameters):
         for observation in self.observations:
             n_obs_params = len(observation.parameters)
             obs_params = parameters[n_params : n_params + n_obs_params]
+            shift = parameters[-1]
             total_loss = total_loss - observation.get_log_likelihood(
-                model, *obs_params, noise_factor=self._noise_factor
+                model, *obs_params, shift, noise_factor=self._noise_factor
```

Comment:
how is any of that necessary? just pass `obs_params`.

Reply:
I think I used that for some tests. Removed